examples/module-dependency-order.js

```diff
@@ -1,6 +1,11 @@
 // example - valid: true
 angular.module('myModule', ['ngAnimate', 'ngRoute', 'app', 'appFilters', 'ui.router']);
 
+// example - valid: true
+// with ES6 syntax:
+// import uiRouter from 'angular-ui-router';
```

Comment:
I tried experimenting with

```
            var eslintTester = new RuleTester();
```

in `./scripts/docs.js` by passing eslint config:

```
            var eslintTester = new RuleTester({ parserOptions: { sourceType: 'module', ecmaVersion: 2016 } });
```

and it helped but it doesn't feel right to set it globally.

So I tried expressing it on example basis:
```
// example - valid: true, eslintOptions: {}
```
with intention to pass `example.eslintOptions` to RuleTester constructor call.

but since it doesn't seem to be possible:

```
        this.rules.forEach(function(rule) {
        if (rule.examples !== undefined) {
            var eslintTester = new RuleTester({ parserOptions: { sourceType: 'module', ecmaVersion: 2016 } });
			//
			// no way to configure tester on per-example basis
			// -----------------------------------------------
			//												 |
			//												 v
            eslintTester.run(rule.ruleName, rule.module, rule.examples);
        }
    });
```

unless we group examples by and create new RuleTester instance for each config?

Anyway. I didn't want to handle it here. Maybe in a different PR. Or maybe you have a better approach?